spec/models/comment_spec.rb

```diff
@@ -133,6 +133,10 @@
     expect(comment.readable_publish_date.include?("'#{last_year}")).to eq(true)
   end
 
+  it "shows the full time in hover date" do
+    expect(comment.hover_publish_date).to eq(comment.created_at.strftime("%B %e %Y %I:%M %P %Z"))
```

Comment:
This expectation is making use of the `hover_publish_date` method's implementation. This will make sure that the method works as it was first implemented.

However, does this scenario guarantee that the output format is the expected?
Should we test against a string representation of the expected output instead?